packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/sku_details_wrapper.dart

```diff
@@ -68,8 +72,17 @@ class SkuDetailsWrapper {
   final String introductoryPrice;
 
   /// [introductoryPrice] in micro-units 990000
-  @JsonKey(name: 'introductoryPriceAmountMicros', defaultValue: '')
-  final String introductoryPriceMicros;
+  @JsonKey(name: 'introductoryPriceAmountMicros', defaultValue: 0)
+  final int introductoryPriceAmountMicros;
+
+  /// String representation of [introductoryPrice] in micro-units 990000
+  @Deprecated('Use `introductoryPriceAmountMicros` instead.')
+  @JsonKey(ignore: true)
+  String get introductoryPriceMicros => _introductoryPriceMicros.isEmpty
+      ? introductoryPriceAmountMicros != 0
+          ? introductoryPriceAmountMicros.toString()
+          : ''
+      : _introductoryPriceMicros;
 
```

Comment:
Needs a no-op, deprecated setter to avoid being a breaking change, per discussion.

Reply:
Oh, right; good call.